librz/bin/format/elf/rz_bin_elf_deprecated.inc

```diff
@@ -235,6 +237,7 @@ bool Elf_(rz_bin_elf_del_rpath)(RzBinFile *bf) {
 	return true;
 }
 
+// DEPRECATED FUNCTION
```

Comment:
You could use `RZ_DEPRECATED` here, also see https://github.com/rizinorg/rizin/pull/865 on how to fish the usage of that function with the compiler